endpoints/openrtb2/amp_auction.go

```diff
@@ -178,6 +178,9 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 		return
 	}
 
+	// upgrade to 2.6 here
+	openrtb_ext.ConvertUpTo26(reqWrapper)
```

Comment:
Should this be moved into `parseAmpRequest` and be called right after the stored request is fetched but before we populate other fields like `req.test`, `req.imp[x].secure`, etc and perform validations?

Reply:
I will move it up to just after we create the request wrapper. Seems to be a good place for it. The functions afterwards only seem to hit non-upconverted fields.